qa/L0_perf_resnet/run_test.sh

```diff
@@ -89,11 +87,13 @@ else
     # Must warmup similar to actual run so that all instances are ready
     # Note: Running extra PA for warmup doesn't make sense for C API since it
     # uses in-process tritonserver which will exit along with this PA process.
+    set +e
```

Comment:
I may be wrong but to enable immediate exit on error we bash syntax is `set -e` and to disable it `set +e`...

Reply:
this command is allowed to fail due to instability from warmup and that is why this is run to warm up the model. We don't want to exit the script as it is allowed to fail